deploy/services-demo/create_team_members.sh

```diff
@@ -2,10 +2,10 @@
 
 set -e
 
-ADMIN_UUID="a09e9521-e14e-4285-ad71-47caa97f4a16"
-TEAM_UUID="9e57a378-0dca-468f-9661-7872f5f1c910"
-BRIG_HOST="http://localhost:8082"
-CSV_FILE="myfile.csv"
+ADMIN_UUID="n/a"
+TEAM_UUID="n/a"
+BRIG_HOST="http://localhost:8080"
```

Comment:
Note that the `8082` default was due to the fact that this is generally the port used by `brig` in integration tests and when [starting services](https://github.com/wireapp/wire-server/blob/develop/services/start-services-only.sh)